src/gp_dag.cpp

```diff
@@ -106,6 +109,87 @@ EigenVectorXd GPDAG::BuildUniformQ() const {
   return q;
 }
 
+Node::NodePtrVec GPDAG::GenerateAllTrees() const {
+  std::vector<Node::NodePtrVec> trees_below(NodeCount());
+
+  auto GetSubtrees = [&trees_below](const GPDAGNode *node,
+                                          Node::NodePtrVec &rotated_subtrees,
+                                          Node::NodePtrVec &ordered_subtrees) {
```

Comment:
I believe we are using "sorted" as the opposite as "rotated", not "ordered".

Reply:
Done.

Reply:
Done.